### ISSUE TITLE
Just a tribute! :)

### ISSUE THREAD

**Issue by xparq** [RickKimball/fabooh#3]
Rick, just wanted to find a way to express my gratitude for you having created this delightful piece of work! :+1: 

(After an abrupt departure and a 3-year pause, I've just found myself here around again, now finally/hopefully getting myself into using your lib in earnest. Also, I was pretty much shocked, to be honest, to find out that the only feedback to this project have come also from me (in 2015)... I completely forgot about that. :) (FYI, at that time I had just ripped your equally wonderful MSP430 GDB bootloader only, as that was all I needed then.))

Thank you!

**Comment by RickKimball** [comment]
Thanks and to be honest, I haven't been doing much to advance the msp430 version of this code.  I doubt I'll add much more to the msp430 stuff.

My focus as of late has been to make the stm32f103 boards work better.   It is hard to beat the price and value you get out of those stm32 "bluepill" boards.

**Comment by xparq** [comment]
"I haven't been doing much to advance the msp430 version of this code. ... My focus ... has been to make the stm32f103 [bluepill] boards work better."

Yes, I noticed, and quite reasonable, too. And, in fact, I had started looking up that board exactly because of your apparent shift of focus (having "bluepill" set by default).
It's an amazing deal, indeed.

The only thing I found little data about is how power-conscious/battery-friendly that board actually is.
What's your experience?
